src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
Yes it would be set "correctly", but now you have two ways of saying the same thing in RippleAPI, via omission or setting to the wildcard counterparty. Just pointing it out.